### PR TITLE
feat: make optional use latest index in landing page

### PR DESCRIPTION
**Pull request by vgelbgras**
In this PR, I added an input to `doc-deploy-dev` and `doc-deploy-stable` so we can skip when needed the step to overwrite the index.html page. 

It will be used in some internal projects.

Tested [here](https://github.com/ansys-internal/glow-engine/actions/runs/9220336792) successfully.

_** Correct the name of the branch used in [PR#492](https://github.com/ansys/actions/pull/492)_

### CODE REVIEW

File: doc-deploy-dev/action.yml
Comment (jorgepiloto) [inline doc-deploy-dev/action.yml:112]:
```suggestion
    description: |
```

File: doc-deploy-stable/action.yml
Comment (jorgepiloto) [inline doc-deploy-stable/action.yml:135]:
```suggestion
    description: |
```

**Review by jorgepiloto — APPROVED** [review]
Approving this. Please, apply the suggestions to ensure that the multi-line description render as expected.

Once merged, I will perform a patch release.

**Comment by vgelbgras** [comment]
> Approving this. Please, apply the suggestions to ensure that the multi-line description render as expected.
> 
> Once merged, I will perform a patch release.

@jorgepiloto could you add me in the contributors? It looks like I am not authorized to merge the PR.

**Review by RobPasMue — APPROVED** [review]
Approving and merging

**Comment by RobPasMue** [comment]
Let's wait for #491 to be merged before doing the patch release

**Pull request merged**
